cypress/integration/instance/InstanceCreation.ts

```diff
@@ -113,6 +114,24 @@ describe("the 'Create a SE instance' Modal", () => {
     });
   });
 
+  // instance with name containing "error-test" causes 4xx response code
+  it("Submit and expect error while creating", () => {
```

Comment:
@jomarko just to be sure, I guess this should run only on the mocked APIs, right? Does it need to be wrapped inside an `onlyOn`?

Reply:
@kelvah thank you for spotting. fixed now, hopefully fef3189

Reply:
Looks good :+1: